client/src/document/on-github.tsx

```diff
@@ -129,46 +102,38 @@ function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
       doc.modified ? new Date(doc.modified).toISOString() : "*date not known*"
     )
     .trim();
-  sp.set("body", body);
-  const maxLength = 50;
-  const titleShort =
-    doc.title.length > maxLength
-      ? `${doc.title.slice(0, maxLength)}…`
-      : doc.title;
-  sp.set("title", `Issue with "${titleShort}": (short summary here please)`);
-
-  const slug = doc.mdn_url.split("/docs/")[1].toLowerCase();
+}
+
+function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
   const { locale } = doc;
+  const url = new URL("");
```

Comment:
This causes a `TypeError`. I believe an empty string is not valid input: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#parameters

Reply:
Tried to fix this with e163c91e9ca6226b0602eb1903292ee79417e621